c11threads.h

```diff
@@ -23,10 +23,21 @@ Main project site: https://github.com/jtsiomb/c11threads
 #ifdef __APPLE__
 /* Darwin doesn't implement timed mutexes currently */
 #define C11THREADS_NO_TIMED_MUTEX
+#include <Availability.h>
+#ifndef __MAC_10_15
+#define C11THREADS_NO_TIMESPEC_GET
+#endif
+#else
+#if __STDC_VERSION__ < 201112L
+#define C11THREADS_NO_TIMESPEC_GET
+#endif
 #endif
```

Comment:
So apple libc for earlier SDK versions defines __STDC_VERSION__ as C11 without providing timespec?

What's in the Availability.h header? Is it needed for __MAC_10_15 to be defined?

I'd rather the last block be condensed to `#elif ... #endif` rather than `#else #if ... #endif #endif`

Reply:
1. Yes o.o
2. I think `Availability.h` is needed for all those macros for versioning: [Availability.h of MacOSX10.15.sdk](https://pastebin.com/tn9Ni4Pw)
3. Can do that...